stable/magic-namespace/values.yaml

```diff
@@ -86,8 +86,8 @@ tiller:
 
 ## Optional additional ServiceAccounts
 serviceAccounts:
-- some-service-account
-- another-service-account
+# - some-service-account
+# - another-service-account
```

Comment:
This doesn't look right. I agree with the change, but I think this is invalid syntax.

In YAML, every key _must_ have a value (same as in JSON). By removing all the elements from the `serviceAccounts` array, you've created a condition where `serviceAccounts` has no value.

To make this work, you'd have to do it like this:

```yaml
## Optional additional ServiceAccounts
serviceAccounts: []
# - some-service-account
# - another-service-account
```

Reply:
I disagree with your assessment, the YAML spec allows for empty assignment, the value is interpreted as null. It would however `range` over null which has the desired outcome but isn't ideal. I'd sooner comment out `serviceAccounts` also and wrap the template in an `if haskey` like `namespaceAttributes`, so that users can simply uncomment and modify lines, not clean up redundant square brackets.

Reply:
You know what... you're right. I have actually been mislead into thinking that no value was invalid _on account of_ the pattern I have advocated for being so ubiquitous throughout the many charts I have used or contributed to.

I think we probably need some repo maintainers to weigh in on what is the preferred convention (if there is one) for dealing with lists that are empty by default.

Reply:
@krancour in the interest of getting this in, happy to lean towards the existing de facto standard and hence have updated my PR accordingly

Reply:
sgtm